gbpservice/nfp/configurator/agents/firewall.py

```diff
@@ -28,7 +28,7 @@
 rest_timeout = [
     cfg.IntOpt(
         'rest_timeout',
-        default=240,
+        default=60,
```

Comment:
Discussed this with Suresh. Looks like we have this timeout parameter for all types of services. We need to make this uniform. In case uniformity is not possible, we have to name them differently.